jobs/pulse3d/manifests/base/job-runner.yaml

```diff
@@ -7,5 +7,9 @@ spec:
   job_queue: pulse3d
   seconds_to_poll_db: 15
   max_num_of_workers: 8
+  ecr_repo: null
+  product_specific:
+    pulse3d_uploads_bucket: null
+    mantarray_logs_bucket: null
```

Comment:
Does anyone have an opinion if they think I should capitalize all of these env variables?